pyproject.toml

```diff
@@ -40,8 +40,6 @@ dependencies = [
     "openpyxl",                                # extra dependency for pandas (excel)
     "opentelemetry-api",
     "opentelemetry-sdk",
-    "opentelemetry-instrumentation-fastapi",
-    "opentelemetry-instrumentation-httpx",
```

Comment:
(these removals can also trigger removal of these packages in komodo)

Reply:
https://github.com/equinor/komodo-releases/pull/7604